dmci/api/app.py

```diff
@@ -39,6 +39,11 @@ def __init__(self):
         def base():
             data = request.get_data()
 
+            max_permitted_size = self._conf.max_permitted_size
+
+            if len(data) > max_permitted_size:
```

Comment:
I was wondering if we should rather set a buffer of max_permitted_size and get the request data in it. The thing we're trying to prevent is that it is quite easy to give a wrong @ to curl and send fex. a netcdf file that will unnecessarily load the server. Currently we still receive the whole data and then discard if it's not acceptable.